html-forms/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <form action="/handling-page" method="post">
+      <label for="mail">Email</label>
+      <br />
```

Comment:
It's fine for now because we haven't learned CSS, but once we learn more about CSS, you should never space items out like this moving forward. It would be considered bad practice. Rather than `<br />`, you could also stuff each `input / label` into a `div` to take advantage of it's default `block` styling